k8s/pkg/apis/longhorn/v1beta2/orphan.go

```diff
@@ -5,12 +5,15 @@ import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 type OrphanType string
 
 const (
-	OrphanTypeReplica = OrphanType("replica")
+	OrphanTypeReplicaData     = OrphanType("replica")
+	OrphanTypeEngineInstance  = OrphanType("engine-instance")
+	OrphanTypeReplicaInstance = OrphanType("replica-instance")
 )
 
 const (
 	OrphanConditionTypeError         = "Error"
 	OrphanConditionTypeDataCleanable = "DataCleanable"
+	OrphanConditionTypeInstanceState = "InstanceState"
```

Comment:
Look like this condition represent InstanceStateTerminated. Should we change the name?

Reply:
Sure, will change to `InstanceExist`, positive if instance manager contains this instance, and record the instance state in reason.